common/lib/xmodule/xmodule/html_module.py

```diff
@@ -75,10 +76,8 @@ def student_view(self, _context):
         return Fragment(self.get_html())
 
     def get_html(self):
-        """
-        When we switch this to an XBlock, we can merge this with student_view,
-        but for now the XModule mixin requires that this method be defined.
-        """
+        # When we switch this to an XBlock, we can merge this with student_view,
```

Comment:
There should still be a docstring here too.  Please add an appropriate docstring in addition to the comment.